attributors/element_style.js

```diff
@@ -1,17 +1,32 @@
 import { StyleAttributor } from 'parchment';
+import { decorateCanAdd, decorateMethodWithKeyName } from './custom_attributor_decorators';
+import { getTablePrefix } from '../formats/table/attributors/key_name_map';
 
-class ElementStyleAttributor extends StyleAttributor {
+export default class ElementStyleAttributor extends StyleAttributor {
   constructor(attrName, keyName, options = { allowedTags: [] }) {
     super(attrName, keyName, options);
 
     this.allowedTags = options.allowedTags ?? [];
   }
 
-  canAdd(node, value) {
-    const isNodeAllowed = this.allowedTags.indexOf(node.tagName) > -1;
+  static keys(node) {
+    const prefix = getTablePrefix(node.tagName);
+    return super.keys.call(this, node).map((keyName) => `${prefix}${keyName}`);
```

Comment:
why do we return keys as they all are table props? It will break formatting inside table with non-table attributes as a color.
Should we return both options?

Reply:
Discussed.
We decided to implement some special fallback logic for attributes/styles like this.